### ISSUE TITLE
Icinga + grafana: Operation timed out after...

### ISSUE THREAD

**Issue by mrPsycho** [Mikesch-mp/icingaweb2-module-grafana#193]
## Expected Behavior
everything is fine

## Current Behavior
graphs don't work: `Cannot fetch graph with curl: 'Operation timed out after 5001 milliseconds with 0 bytes received'.`

## Your Environment
config.ini:
```
[grafana]
version=1
username = "admin"
host = "monitoring.server.com/grafana"
protocol = "https"
password = "superBRUTAL"
height = "280"
;width = "640"
;timerange = "3h"
;timerangeAll = "1M/M"
enableLink = "yes"
defaultorgid = "1"
defaultdashboard = "icinga2-influxdb-host"
defaultdashboarduid = "p6Zkhnhmk"
defaultdashboardpanelid = "1"
shadows = "false"
;datasource = "influxdb"
;defaultdashboardstore = "db"
accessmode = "proxy"
authentication = "basic"
timeout = "5"
directrefresh = "no"
;usepublic = "no"
;publichost = "otherhost:3000"
;publicprotocol = "http"
;custvardisable = "idontwanttoseeagraph"
debug = "1"
ssl_verifypeer = "0"
ssl_verifyhost = "0"
```

grafana: 5.3.4
icingaweb2-module-grafana: 1.3.2
icingaweb2: 2.6.1

grafana installed on https://monitoring.server.com/grafana

on icinga i see this: 
```
Performance Graph Debug
Access mode: proxy
Authentication type: basic
Protocol: https
Grafana Host: monitoring.dz.ru/grafana
Dashboard UID: p6Zkhnhmk
Dashboard Name: icinga2-influxdb-host
Panel ID: 1
Organization ID: 1
Theme: light
Timerange: now-6h
Timerangeto: now
Height: 280
Width: 640
Custom Variables: 
Graph URL: https://monitoring.server.com/grafana/render/d-solo/p6Zkhnhmk/icinga2-influxdb-host?var-hostname=node.server.com&var-service=hostalive&var-command=hostalive&panelId=1&orgId=1&width=640&height=280&theme=light&from=now-6h&to=now
Disable graph custom variable: grafana_graph_disable
Graph config custom variable: grafana_graph_config
Shadows: Yes
SSL Verify Peer: No
SSL Verify Host: No
```

on `grafana` i have this working dashboard: https://monitoring.server.com/grafana/d/p6Zkhnhmk/icinga2-influxdb-host?orgId=1&var-hostname=node.server.com&var-service=All
and it shows all graphs. 

where could be the problem?

**Comment by Mikesch-mp** [comment]
Hello @mrPsycho ,

did you try to raise the timeout to 10 or more secs? Depending on the dasboard the renderer takes more time.

**Comment by mrPsycho** [comment]
@Mikesch-mp hmmmm.... ive change variable `timeout` to 15, but it still shows:
`Cannot fetch graph with curl: 'Operation timed out after 5001 milliseconds with 0 bytes received'.`

**Comment by unipanic** [comment]
I'm having this same issue and increasing the timeout has not helped. If I click on the graph, it takes me to the correct graph in Grafana and there is data there. I'm following this in hopes of a fix.

**Comment by Mikesch-mp** [comment]
@mrPsycho 

Please use the provided icinga2-default dasboard and not the `icinga2-influxdb-host`. I wrote on the Grafana site "o not use this with the icingaweb2-module-grafana!" ;)
You can always test the renderer by opening the URL from the debug output

**Comment by Mikesch-mp** [comment]
@unipanic you use trhe same dashboard as default dashboard? if not please open a seperate issue

**Comment by unipanic** [comment]
No I use my own custom built dashboard cause I monitor a bunch of custom items, so I'll open a new issue. Thank you!

**Comment by kogaba** [comment]
> Graph URL: https://monitoring.server.com/grafana/render/d-solo/p6Zkhnhmk/icinga2-influxdb-host?var-hostname=node.server.com&var-service=hostalive&var-command=hostalive&panelId=1&orgId=1&width=640&height=280&theme=light&from=now-6h&to=now

I think your custom build dashboard don't have panelid=1
I don't know other solution to have panelid 1 other than importing the default json from module-grafana to grafana.
If you want to use custom panels for your services, you have to add the services one by one in icingaweb interface -> configuration -> grafana graphs
(you can get the panel ids from grafana -> json view)

**Comment by Mikesch-mp** [comment]
@mrPsycho 

can you please use the latest release and make sure that the panelid = 1 on your dashboard. i think thats maybe the error.

**Comment by martinadolfi** [comment]
I had the same issue some days ago, it was about the `d-solo` part in the render url that changed to `dashboard-solo` perhaps in some newer version of grafana. I updated the module and it worked right away. 
Then I upgraded my other instances of icinga with the newest module, and they stopped working for the same reason ;) , so I think that the version of the module should match the grafana version so the url works properly. I think it'll also work if I upgraded the grafana version... but didn't try, I just downgraded the module again.
